sdk/servicebus/test-resources.json

```diff
@@ -174,52 +174,26 @@
       }
     }
   ],
-  "functions": [
-    {
-      "namespace": "connectionString",
-      "members": {
-        "create": {
-          "parameters": [
-            {
-              "name": "accountName",
-              "type": "string"
-            },
-            {
-              "name": "accountKey",
-              "type": "string"
-            },
-            {
-              "name": "blobEndpoint",
-              "type": "string"
-            }
-          ],
-          "output": {
-            "type": "string",
-            "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', parameters('accountName'), ';AccountKey=', parameters('accountKey'), ';BlobEndpoint=', parameters('blobEndpoint'))]"
-          }
-        }
-      }
-    }
-  ],
   "outputs": {
+    "SERVICEBUS_FULLY_QUALIFIED_NAMESPACE": {
+      "type": "string",
+      "value": "[concat(variables('serviceBusNamespace'), '.servicebus.windows.net')]"
+    },
     "SERVICEBUS_CONNECTION_STRING": {
       "type": "string",
       "value": "[listkeys(variables('authRuleResourceId'), variables('sbVersion')).primaryConnectionString]"
     },
-    "SERVICEBUS_SECONDARY_NAMESPACE_CONNECTION_STRING": {
+    "SERVICEBUS_SECONDARY_FULLY_QUALIFIED_NAMESPACE": {
       "type": "string",
-      "value": "[listkeys(variables('authRuleSecondaryResourceId'), variables('sbVersion')).primaryConnectionString]"
+      "value": "[concat(variables('serviceBusSecondaryNamespace'), '.servicebus.windows.net')]"
```

Comment:
Incorrect for some sovereign clouds that use different domains.  We can't assume here, we should parse these from the resource.